packages/backend/src/config/parseConfig.ts

```diff
@@ -705,6 +707,8 @@ export const parseConfig = (): LightdashConfig => {
         headlessBrowser: {
             port: process.env.HEADLESS_BROWSER_PORT,
             host: process.env.HEADLESS_BROWSER_HOST,
+            internalLightdashHost:
+                process.env.INTERNAL_LIGHTDASH_HOST || siteUrl,
```

Comment:
yes. There was a duplicate [issue](https://github.com/lightdash/lightdash/issues/11186) where the user wanted to avoid these requests from leaving their infra (aka not going to public internet)

Reply:
Added to docs